frontend/packages/dev-console/src/components/import/devfile/DevfileStrategySection.tsx

```diff
@@ -47,11 +49,25 @@ const DevfileStrategySection: React.FC = () => {
       setFieldValue('devfile.devfileHasError', true);
       setValidated(ValidatedOptions.error);
     } else {
+      fireTelemetryEvent('Download Devfile from Git', {
+        client: 'openshift-console',
+        devfileName: selectedSample.name,
```

Comment:
The `DevfileStrategySection` is used not just in the samples flow but also the import from git flow and therefore I believe that there won't always be a `selectedSample` available which would cause this to throw an error.

cc @rohitkrai03

Reply:
I tested this out in the Git import page and did not see any console errors.  I did not see this event being sent to Segment either but I can add a null check.

Reply:
You won't see a segment event unless you install a telemetry listener that sends the event to segment. This is done with the telemetry plugin which is installed on dev sandbox.

Reply:
I installed the dynamic demo plugin to get this working locally